packages/edit-post/src/components/keyboard-shortcut-help-modal/shortcut.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { castArray } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+const mapKeyCombination = ( keyCombination ) => keyCombination.map( ( character, index ) => {
```

Comment:
It's a bit non-conventional. I might have expected this would be a child component, not a mapping function.

Reply:
Yes, this is not a new function, I just moved it from another file. I'll see if I can do a small refactor.